partcad/src/partcad/cache_hash.py

```
@@ -0,0 +1,121 @@
+
+class CacheHash:
+    def __init__(self, name: str, algo="md5", hasher=None):
+        self.name = name
+        self.is_empty = True
+        self.is_used = False
+        if not user_config.cache:
+            # Caching is disabled, no initialization needed
+            self.hasher = None
+            return
+
+        if hasher != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)